timers.js

```diff
@@ -1,4 +1,36 @@
+var startButton = document.getElementById("start");
+var timerText = document.getElementById("timer");
+var seconds = 0;
+var resetButton = document.getElementById("reset");
+var pauseButton = document.getElementById("pause");
+var timeInterval;
+var paused = false;
 
+startButton.addEventListener("click", updateTime);
+resetButton.addEventListener("click", resetTime);
+pauseButton.addEventListener("click", pauseTime);
 
+function updateTime() {
+  timeInterval = setInterval(startTime,1000);
+}
```

Comment:
so the return value of setInterval is an integer that represents the ID of the setInterval. So when if you were to call this more than once, `timeInterval` will only store the most recent `setInterval()` ID. Thereby when you clear the interval only the most recent interval will be cleared. So you would need to incorporate a boolean value to adjust for this ie something like this:

```js
var clockRunning = false
function updateTime(){
  if(!clockRunning){
    timeInterval = setInterval(startTime, 1000);
    clockRunning = true
  } else {
    clockRunning = false
  }
}

```